<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogNewInternal</class>
 <widget class="QDialog" name="DialogNewInternal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>390</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>411</width>
    <height>390</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>411</width>
    <height>390</height>
   </size>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Nouvelle variable interne...</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/img/icon_bool_on.png</normaloff>:/img/icon_bool_on.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>391</width>
     <height>27</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>Create a new Internal variable</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Item Name :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>351</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_error_empty">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>370</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>146</red>
          <green>145</green>
          <blue>144</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="text">
     <string>Please enter a name for this item..</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Value :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="value_text">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>351</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_bool">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>90</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Boolean</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_int">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>70</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Integer</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_text">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>70</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Text</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="value_bool">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>152</y>
      <width>97</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Activated</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="value_int">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>90</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-9999</number>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_visible">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>280</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Visible</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_save">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>330</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Automatically save the value</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_rw">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>330</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit Mode</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogNewInternal</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
